pkgs/applications/graphics/freecad/default.nix

```diff
@@ -9,13 +9,13 @@ let
   pythonPackages = python3Packages;
 in mkDerivation rec {
   pname = "freecad";
-  version = "0.18.4";
+  version = "unstable-2020-09-25";
```

Comment:
Version must start with a digit, "unstable" should be part of pname: https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

Reply:
Done.